src/web-map.js

```diff
@@ -76,26 +76,48 @@ export class WebMap extends HTMLMapElement {
   constructor() {
     // Always call super first in constructor
     super();
-    this.style.display = "block";
```

Comment:
Did this get moved?

Reply:
Yes, moved to [here](https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/108/files#diff-d6164714af1bf6c21f54c11ca3e1a588R97). However it is now `display: inline-block` as it aligns better, quite literally, with `<iframe>`/`<video>` if you have them side by side  (they're `display: inline` by default, the closest we can get to native with the map element is `display: inline-block`).